### ISSUE TITLE
[Bug] "Left group" notification spam.

### ISSUE THREAD

**Issue by WindmillArt** [vrcx-team/VRCX#790]
As of v2024.05.09, VRCX spams me with "left group" notifications every few minutes for every one of my own groups.
Mind you, I'm not leaving my groups in the first place nor done anything with the groups in general. It just tells me I've left them over and over again.

Rolled back to v2024.04.23 for now.

**Comment by OctoFloofy** [comment]
What happens if you try the recent Nightly version? Had that issue when it first got added but since then it got fixed afaik

**Comment by Natsumi-sama** [comment]
This update is after all those fixes, are you able to open the console with `Ctrl + Shift + I` and screenshot any errors, can DM me on Discord if you'd like to keep the info private `NatsumiDa`

**Comment by WindmillArt** [comment]
Nightly 2024-05-15T10.05-523ac66 yields same result. Upon launching and waiting for a minute I get notification of me leaving my groups again. Console prints nothing when this happens.
I waited around to see if I get more of these notifications, but they only happen after launch.

![image](https://github.com/vrcx-team/VRCX/assets/12586882/1ba53ee0-f8ea-4cfd-bd98-88f7c34645ca)

Switched back to stable v2024.05.09 and it's the same deal. About a minute after launch, I get a notification of me having left my groups. Nothing in the console there either.

Time between repeat notifications seems to be random, though. Initially I got them every 5 or so minutes, but now they repeat after around 15 minutes.
